oauth2client/contrib/gce.py

```diff
@@ -55,8 +52,7 @@ class AppAssertionCredentials(AssertionCredentials):
     information to generate and refresh its own access tokens.
```

Comment:
Yeah, but I'm wondering if that actually changed any real behavior. Hence trying it out with google-api-python-client